### ISSUE TITLE
The work you've done is really impressive! Is it possible to implement it in Fallout 4?

### ISSUE THREAD

**Issue by Ssadeness** [ArranzCNL/ImprovedCameraSE#116]
As of 2023, the modding ini is still being used to adjust the first person camera in Fallout 4。

**Comment by Ssadeness** [comment]
At the same time, everyone seemed to think it was impossible to achieve and that no one had ever tried to do it. It's no exaggeration to say that if there is one person in the world who could make it happen, that person could only be you!

**Comment by ArranzCNL** [comment]
Short answer: Yes (Done it back late 2019 got the model drawing - wasn't in the correct place, or rather it was just ontop of you!)

Long answer:
There is one doom/gloom approaching which is the update, everyone should update imho. I'd imagine it might be abit more severe than SkyrimSE 1.5.39 to 1.6.x (not as bad as SkyrimLE (x86) to SkyrimSE (x64)).

I won't have the time in a speedily manner to get up and running, although I want too. SkyrimSE is my priority and once the next update is released there are 2 other features I want to implement, 1 is external, other is internal both are a huge benifit imho to IC and maybe the external to the community as a whole. Which would also benifit the FO4 port!

Once the source to the NG release is available and someone is interested in porting I can help, just limited. I apologise for sources not being available at the time of writing as I know someone would have tried porting to CLib before I had a chance and no doubt just done a 1:1 which wouldn't be correct.

TLDR:
I want too but can also help other(s) out if they want too once NG sources are available whilst I work on the other feature(s) mentioned.
